webrtc/RTCPeerConnection-setRemoteDescription.html

```diff
@@ -47,6 +47,43 @@
       assert_unreached('Error ' + e.name + ': ' + e.message);
     }));
   }, 'Triggers ontrack when called with a remote description and the MSID is parsed.');
+
+  promise_test(() => {
+    const pc = new RTCPeerConnection();
+    return pc.setRemoteDescription(
+      {'type': 'offer', 'sdp': 'bogus'})
+    .then(() => {
+      assert_unreached('Bogus SDP should not be accepted');
+    })
+    .catch(e => {
+      assert_equals(e.name, 'InvalidAccessError');
```

Comment:
It looks like JSEP uses "SessionDescription object" to mean "RTCSessionDescription object/instance", but there still isn't anything that I can see that explains that "content of *description*" is actually the sdp member.